stdlib/gc.mli

```diff
@@ -157,16 +160,19 @@ type control =
        compaction is triggered at the end of each major GC cycle
        (this setting is intended for testing purposes only).
        If [max_overhead >= 1000000], compaction is never triggered.
-       Default: 500. *)
+
+       This field is currently not available in OCaml 5: the field value is
+       always [0]. *)
 
     stack_limit : int;
     (** The maximum size of the fiber stacks (in words).
-       Default: 1024k. *)
+       Default: 128M. *)
```

Comment:
Where did you find this limit? It looks to me that this field is completely unused in OCaml 5.

Reply:
It is used, but not so easy to grep due to the FFI boundary in `runtime/gc_ctrl.c`. On the C side you have to look for `caml_max_stack_wsize`.

Reply:
Ah, thanks for the pointer.